pages/database-management/authentication-and-authorization/auth-system-integrations.mdx

```diff
@@ -9,42 +9,59 @@ import { Tabs, Tab } from 'nextra/components'
 
 # Auth system integrations (Enterprise)
 
-Memgraph supports authentication and (optional) authorization using a **custom
-built external auth module**. The two supported operation modes are:
-- [DEPRECATED] authentication only (username/password verification)
-- authentication and authorization (username/password verification and user to
-  role mapping)
-
-When a user connects to Memgraph, the database will forward the user's supplied
-username and password to the external auth module and wait for it to deliver
-the authentication and authorization verdict back to the database. Based on
-the returned verdict, Memgraph will either close the connection to the
-connected user or will allow the connection and associate the connection 
-with the appropriate role.
-
-When Memgraph is switched to use the external auth module for authentication
-its internal users are automatically disabled. All users are authenticated only
-using the module, existing local users are ignored. In order to authorize query 
-execution, the auth module must return a role associated with the user. This 
-role must be registered locally on Memgraph.
+Memgraph supports authentication and authorization using **external auth
+modules**. There’s built-in support for basic (username + password) auth via
+LDAP, and single-sign on over the SAML and OIDC protocols.
+
+When a user is connecting, Memgraph will forward the user’s supplied
+credentials (username + password, or scheme and auth response) to the external
+auth module and retrieve its verdict. Based on that, Memgraph will let the user
+connect, or in case of failure, close the connection.
```

Comment:
```suggestion
Memgraph supports authentication and authorization using external auth modules. It includes built-in support for basic (username and password) authentication via LDAP, as well as single sign-on over the SAML and OIDC protocols.

When a user connects, Memgraph forwards the supplied credentials (username and password, or scheme and auth response) to the external auth module and retrieves its verdict. Based on this verdict, Memgraph will either allow the user to connect or, in case of failure, close the connection.
```

Reply:
Applied (except the tense changes in `When a user connects` because they imply connection goes through before SSO)